packages/react-day-picker/src/types/DayPicker.ts

```diff
@@ -229,6 +229,8 @@ export interface DayPickerProps {
   formatters?: Partial<Formatters>;
   // #endregion
 
+  initialFocusOnDay?: boolean;
```

Comment:
I'm not convinced by the name of this prop – i'd expect a "day" to "focus on", not a boolean. Maybe 

```suggestion
  /** 
  * When set, DayPicker will focus on today, or the first selected day if set.
  **/
  focus?: boolean;
```

(not sure the description makes sense, please help(

```ts
<DayPicker mode="single" focus /> // vs.
<DayPicker initialFocusOnDay mode="single" />
```

alternatives considered: `hasFocus`, `hasInitialFocus`

Reply:
How about `initialDayFocus` or `initialFocus`?

Reply:
@tjfred35 `initialFocus` sounds perfect, thanks for looking it up!